### PR TITLE
Juarezds

### PR DESCRIPTION
**Pull request by iamdez99**

### CODE REVIEW

File: src/test/java/ServiceNow/CHARMS/StepsImplementation/RasScenario1StepsImpl.java [hunk 1]
```diff
@@ -527,7 +538,8 @@ public void rasScreenerSubmissionScenario1() {
         rasScenario1StepsImpl.clickOnScreenerNextButton();
 
         try {
-            CommonUtils.waitForVisibility(myRASSurveyPage.dynamicTopText(126));;
+            CommonUtils.waitForVisibility(myRASSurveyPage.dynamicTopText(126));
+            ;
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/CHARMS/StepsImplementation/RasScenario1StepsImpl.java:542]: I  believe there is an extra semi colon here.

File: src/test/java/ServiceNow/CHARMS/Features/IIQ/IIQ Submission.feature [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Feature: RAS Screener Scenarios
   Description: This feature file contains scenarios which submit the IIQ Form
 
-  @bucurgb @myRasStudy @IiqFormsOnly
+  @bucurgb @myRasStudy @IIQ
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/CHARMS/Features/IIQ/IIQ Submission.feature:4]: It would be nice to add the user story id if available.

**Review by hamidnesar — CHANGES_REQUESTED** [review]
Please remove the extra semi colon on line 542 of RasScenario1StepsImpl.java

**Review by georgeb79 — APPROVED** [review]

**Review by alenaminsk — APPROVED** [review]

File: src/test/java/ServiceNow/CHARMS/StepsImplementation/RasScenario1StepsImpl.java (hunk 1 above)
Comment (iamdez99) [reply src/test/java/ServiceNow/CHARMS/StepsImplementation/RasScenario1StepsImpl.java]: Fixed

**Review by hamidnesar — APPROVED** [review]

**Review by puzikovam2 — APPROVED** [review]
reviewed by puzikovam2

**Pull request merged**